tests/tfengine/integration_test.go

```diff
@@ -21,6 +21,11 @@
 //    - `roles/resourcemanager.projectCreator` on the folder
 //    - `roles/resourcemanager.projectDeleter` on the folder
 //    - `roles/resourcemanager.folderAdmin` on the folder
+//    - `roles/cloudbuild.builds.editor` on the folder
+//    - `roles/iam.serviceAccountAdmin` on the folder
+//    - `roles/iam.serviceAccountUser` on the folder
+//    - `roles/source.admin` on the folder
+//    - `roles/storage.admin` on the folder
```

Comment:
Please add a comment for why these roles are needed.

Reply:
I believe `roles/serviceusage.serviceUsageAdmin` and `roles/resourcemanager.lienModifier` roles are needed for the same reason. So please group them together.

Reply:
Do you mean add a comment in the code?

Reply:
Yes. Something like

```
// Prerequisites:
//  - The terraform binary must be available in $PATH.
//  - The environment variable RUN_INTEGRATION_TEST must be set to "true".
//  - The environment variables BILLING_ACCOUNT, FOLDER_ID and DOMAIN must be set.
//  - The runner (e.g. authenticated local user or service account) must have
//    the following roles:
//   Roles as listed under https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/tree/master/docs/tfengine#prerequisites:
//    - `roles/resourcemanager.projectCreator` on the folder
//    - `roles/resourcemanager.folderAdmin` on the folder
//    - `roles/compute.xpnAdmin` on the folder
//    - `roles/billing.admin` on the billing account
//    Additional roles needed because ...
//    - `roles/resourcemanager.projectDeleter` on the folder
//    - `roles/cloudbuild.builds.editor` on the folder
//    - `roles/iam.serviceAccountAdmin` on the folder
//    - `roles/iam.serviceAccountUser` on the folder
//    - `roles/source.admin` on the folder
//    - `roles/storage.admin` on the folder
//    - `roles/serviceusage.serviceUsageAdmin` on the folder
//    - `roles/resourcemanager.lienModifier` on the folder
```

Reply:
Added additional context for roles.